README.md

```diff
@@ -146,17 +146,18 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
         .shard_scheme(scheme)
         .build()
         .await?;
+    let cluster = Arc::new(cluster);
 
     // Start up the cluster.
-    let cluster_spawn = cluster.clone();
+    let cluster_spawn = Arc::clone(&cluster);
```

Comment:
I don't think that there's much utility in using fully qualified syntax for `clone()`, the user already knowns their type + these types don't implement clone by default.
```suggestion
    let cluster_spawn = cluster.clone();
```

Reply:
It's idiomatic to use `Arc::clone` to make it clear that the value itself isn't being cloned. Additionally, we use this standard elsewhere in the library.

Reply:
> It's idiomatic to use `Arc::clone` to make it clear that the value itself isn't being cloned.
The arc docs mention that both styles are used, without specifying a preference.

 I'll yield since this is purely a stylistic nit